Cloud docs 3.0 overhaul (#13889)
PrefectHQ/prefect: docs/3.0rc/cloud/troubleshoot-cloud.mdx

- Are you logging into Prefect Cloud 3? Prefect Cloud 1, Prefect Cloud 2, and Prefect Cloud 3 use separate accounts. 
Make sure to use the right Prefect Cloud 3 URL: \<https://app.prefect.cloud/>
- Do you already have a Prefect Cloud account? If you’re having difficulty accepting an invitation, 
try creating an account first using the email associated with the invitation, then accept the invitation.
- Are you using a single sign-on (SSO) provider, social authentication (Google, Microsoft, or GitHub) or just using an emailed link?

Other tips to help with login difficulties:

- Hard refresh your browser with Cmd+Shift+R.
- Try in a different browser. We actively test against the following browsers:
  - Chrome
  - Edge
  - Firefox
  - Safari
- Clear recent browser history/cookies

None of this worked?

Email us at \<help@prefect.io> and provide answers to the questions above in your email to make it faster to troubleshoot 
and unblock you. Make sure you add the email address you used to attempt to log in, your Prefect Cloud account name, and, if applicable, 
the organization it belongs to.

# General troubleshooting

> If you experience an error with Prefect, there are many paths to understanding and resolving it. 

Reviewer: ```suggestion
```

The first troubleshooting step is to confirm that you are running the latest version of Prefect.
If you are not, upgrade (see below) to the latest version, since the issue may have already been fixed. 
Beyond that, there are several categories of errors:

* The issue may be in your flow code, in which case you should carefully read the [logs](#logs).
* The issue could be with how you are authenticated, and whether or not you are connected to [Cloud](#cloud).
* The issue might have to do with how your code is [executed](#execution).

## Upgrade

> Prefect is constantly evolving by adding new features and fixing bugs. It's likely that a patch has already been 

Reviewer: ```suggestion
Prefect is constantly evolving by adding new features and fixing bugs. A patch may have already been 
```
